### DISCUSSION TITLE
The package does not detect Spanish translations

### DISCUSSION THREAD

**Discussion by aquinoaldair** [MarcinOrlowski/laravel-api-response-builder#223] [Q&A]
I use lang "es" in my config app but the package does not detect.  

The response always is:

```
    "success": true,
    "code": 200,
    "locale": "es",
    "message": "response-builder::builder.ok",
    "data":  null
```
however I have a builder.php in the next route : resources/lang/es/builder.php with spanish text.

**Comment by MarcinOrlowski** [comment]
Language support is provided by Laravel. There's basically nothing special in `ResponseBuilder` but just use of `Lang`, Are you sure you got that correctly configured?
